apps/daimo-mobile/src/logic/linkStatus.ts

```diff
@@ -1,14 +1,29 @@
-import { DaimoLink, formatDaimoLink } from "@daimo/common";
+import {
+  DaimoLink,
+  formatDaimoLink,
+  getInviteStatus,
+  stripSeedFromNoteLink,
+} from "@daimo/common";
 import { DaimoChain } from "@daimo/contract";
 
+import { getAccountManager } from "./accountManager";
 import { env } from "./env";
 
 export function useFetchLinkStatus(
   link: DaimoLink | undefined,
   daimoChain: DaimoChain
 ) {
-  const url = link ? formatDaimoLink(link) : undefined;
-  const rpcHook = env(daimoChain).rpcHook;
-  const ret = rpcHook.getLinkStatus.useQuery({ url: url! }, { enabled: !!url });
-  return ret;
+  const sanitizedLink = link && stripSeedFromNoteLink(link);
+  const url = sanitizedLink && formatDaimoLink(sanitizedLink);
+  const { rpcHook } = env(daimoChain);
+  return rpcHook.getLinkStatus.useQuery({ url: url! }, { enabled: !!url });
+}
+
+export async function fetchInviteLinkStatus(inviteLink: DaimoLink | undefined) {
+  const daimoChain = getAccountManager().getDaimoChain();
```

Comment:
could pass as arg to fix failing test import

Reply:
will do. as part of #620 we can separate testnet vs mainnet links onto separate domains (> we already have test.daimo.xyz ) and simplify a lot of code